### DISCUSSION TITLE
Pass onclick to specific cell of a row

### DISCUSSION THREAD

**Discussion by psammy171** [TanStack/table#5126] [Q&A]
When I am defining columns, I want to receive a onClick function. When using Table in the component, I have function ```onEdit```. I want to pass this onEdit function to onClick attribute in specific column only. Can anyone show me how to achieve this

**Comment by mdrahiem** [comment]
If you are using v8, then you can pass it in the columns's meta. 

```
const table = useReactTable({
    data: data ?? [],
    columns
  })
  
 columnHelper.accessor('rating', {
    header: () => <div>Rating</div>,
    enableSorting: true,
    cell: cell => {
       return <div onClick={() => setShowModal(true)}>cell.getValue()</div>
    },
    meta: {
      setShowModal // this is from the useState
    }
  })
  ```
